exchanges/kraken/kraken.go

```diff
@@ -350,14 +350,16 @@ func (k *Kraken) GetTrades(symbol string) ([]RecentTrades, error) {
 		return recentTrades, errors.New("unexpected data received")
 	}
 
-	trades, ok := tradeInfo[symbol].([]interface{})
+	var trades []interface{}
+	trades, ok = tradeInfo[symbol].([]interface{})
```

Comment:
Not your issue but this can slightly obfuscate an issue where a symbol won't be matched in the lookup table and data above might panic if we get an issue. 

```
data, ok := result.(map[string]interface{})
	if !ok {
		return recentTrades, errors.New("unexpected data received")
	}

	resulty, ok := data["result"]
	if !ok {
		return recentTrades, errors.New("unexpected data received")
	}

	tradeInfo, ok := resulty.(map[string]interface{})
	if !ok {
		return recentTrades, errors.New("unexpected data received")
	}

	tradeyTrades, ok := tradeInfo[symbol]
	if err != nil {
		return recentTrades, fmt.Errorf("symbol [%s] not found in returned data", symbol)
	}

	var trades []interface{}
	trades, ok = tradeyTrades.([]interface{})
	if !ok {
		return recentTrades, errors.New("unexpected data received")
	}
```